app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt

```diff
@@ -34,4 +35,43 @@ class WelcomeActivityTest {
                     .check(matches(not(isDisplayed())))
         }
     }
+
+    @Test
+    fun ifSkipButtonClicked() {
+        if (!ConfigUtils.isBetaFlavour()) {
+            onView(withId(R.id.finishTutorialButton))
+                    .perform(ViewActions.click())
+            assert(activityRule.activity.isDestroyed)
+        }
+    }
+
+    @Test
+    fun ifSwiped() {
+        if (!ConfigUtils.isBetaFlavour()) {
```

Comment:
I think this test should work whether we are on beta or not so we can remove this if condition.